test/Microsoft.TestPlatform.TestUtilities/Workspace.cs

```diff
@@ -8,27 +8,19 @@ namespace Microsoft.TestPlatform.TestUtilities
 {
     public class Workspace : IDisposable
```

Comment:
I know there is no change, but imho calling this TempDirectory and giving it a doc comment that says it represents a dictionary in temporary path that attempts to delete itself once disposed would be nice. So we know what it is when we hover over it.

Reply:
I think workspace is quite generic and is common name but I am fine either way. 

@MarcoRossignoli What's your opinion?

Reply:
Not a strong opinion, my 2c I think that `Workspace` is one layer above the directory concept...workspace contains also some storage but can contains also other stuff.

Reply:
I was thinking in the same direction as you Marco. Workspace simply seemed like a distracting concept that I should grasp, rather than just a helper for creating and cleaning temp directories.

Reply:
(I like a lot the way we're doing inside code coverage repo and used also around, we have a way to create an solution/project object model add source files update project file restore/build/run etc...everything inside that "workspace". Looks more powerful that use test assets build to me.)

Reply:
That is next step. Right now we need to get our tests running in parallel. And then we can look at tightening the developer loop.